plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/enums/CoreArtifactTypes.java

```diff
@@ -380,7 +380,8 @@ public interface CoreArtifactTypes {
 
    ArtifactTypeToken NativeArtifact = osee.add(osee.artifactType(20L, "Native Artifact", true, Artifact)
       .zeroOrOne(Extension)
-      .zeroOrOne(NativeContent));
+      .zeroOrOne(NativeContent)
+      .zeroOrOne(MicrosoftOfficeApplication));
```

Comment:
Would we ever use this for another application?  Might want just OsApplication?  Again, just a thought, don't have to change it.